cmd/archeio/internal/e2e/e2e_containerd_linux_test.go

```diff
@@ -94,6 +94,19 @@ func testE2EContainerdPull(t *testing.T, containerdVersion string) {
 		}
 	})
 
+	// wait for containerd to be ready
+	containerdReady := false
+	for i := 0; i < 5; i++ {
+		if err := exec.Command("ctr", "--address="+socketAddress, "version").Run(); err == nil {
```

Comment:
`ctr version` prints both the client and server version like `kubectl version`, it will hang if the server is not available.